### PR TITLE
feat: allow skipping topic creation

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Allow skipping topic creation

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [x] This PR requires documentation updates.

### CODE REVIEW

**Review by fengjiachun — APPROVED** [review]
LGTM

File: config/metasrv.example.toml [hunk 1]
```diff
@@ -99,7 +99,12 @@ provider = "raft_engine"
 ## The broker endpoints of the Kafka cluster.
 broker_endpoints = ["127.0.0.1:9092"]
 
-## Number of topics to be created upon start.
+## Create topics for WAL.
+## Set to `true` to automatically create topics for WAL.
+## Otherwise, use topics named `topic_name_prefix_[0..num_topics)`
+create_topic = true
```
Comment (killme2008) [inline config/metasrv.example.toml:105]: `auto_create_topics` would be better.

Comment (WenyXu) [reply config/metasrv.example.toml]: Resolved in [b324e5c](https://github.com/GreptimeTeam/greptimedb/pull/4616/commits/b324e5c44b613203d64cd0966c0d927fd74beb8b)

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**